feature_tests/steps/_1_setup.py

```diff
@@ -55,9 +56,13 @@ def registered_in_system(
         if with_without_any == "with"
         else []
     )
+
+    org_id, org_id_extension = split_custodian_id(test_config.actor_context.org_id)
```

Comment:
Is this correct? I think those come from splitting the producer_id because the custodian is the org_id in this case